runtime/mandala/src/lib.rs

```diff
@@ -1146,7 +1146,7 @@ parameter_types! {
 	];
 	pub const ExtendedProvisioningBlocks: BlockNumber = 2 * DAYS;
 	pub const TradingPathLimit: u32 = 4;
-	pub AlternativeSwapPathJointList: Vec<Vec<CurrencyId>> = vec![vec![GetStakingCurrencyId::get()]];
+	pub AlternativeSwapPathJointList: Vec<Vec<CurrencyId>> = vec![vec![GetStakingCurrencyId::get()], vec![GetLiquidCurrencyId::get()]];
```

Comment:
Could feature gate the benchmarks instead, but figured mandala mirroring acala/karura is better anyways